api/specs/webserver/openapi.yaml

```diff
@@ -236,6 +236,12 @@ paths:
   /projects/{project_id}/nodes/-/services:access:
     $ref: "./openapi-projects-pydantic.yaml#/paths/~1projects~1{project_id}~1nodes~1-~1services:access"
 
+  /projects/{project_id}/nodes/-/homepage:
```

Comment:
Would it be better to use another word than the _homepage_? I was thinking homepage is on the project level. I open a homepage of a project correct? Maybe here it is more something like:
`/projects/{project_id}/nodes/-/preview`

Reply:
I changed to your suggestion! thx